client/components/ItemActionConfirmation/forms/cancelCoverageForm.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {get, cloneDeep, isEmpty} from 'lodash';
+
+import * as selectors from '../../../selectors';
+import {formProfile} from '../../../validators';
+import {gettext} from '../../../utils';
+import {PLANNING, WORKFLOW_STATE} from '../../../constants';
+
+import {Row} from '../../UI/Preview';
+import {TextAreaInput} from '../../UI/Form';
+import '../style.scss';
+
+export class CancelCoverageComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {reason: '', errors: {}};
+
+        this.onReasonChange = this.onReasonChange.bind(this);
+        this.formProfile = this.props.coverageCancelProfile;
```

Comment:
I know the `formProfile` won't change, but it is always best practice to access it via props instead of an instance local variable.